yarn.lock

```diff
@@ -19096,10 +19096,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"http-cache-semantics@npm:^4.0.0, http-cache-semantics@npm:^4.1.0":
-  version: 4.1.0
-  resolution: "http-cache-semantics@npm:4.1.0"
-  checksum: 974de94a81c5474be07f269f9fd8383e92ebb5a448208223bfb39e172a9dbc26feff250192ecc23b9593b3f92098e010406b0f24bd4d588d631f80214648ed42
+"http-cache-semantics@npm:^4.1.1":
```

Comment:
Here is the required update:
```suggestion
"http-cache-semantics@npm:^4.0.0, http-cache-semantics@npm:^4.1.0":
```

Reply:
Hmm, GitHub isn't letting me commit this suggestion. Keeps giving an error about the diff being updated. Seems like a bug. Oh well.

Reply:
done in latest commit